docker/cassandra/default-config/jvm.options

```diff
@@ -0,0 +1,42 @@
+-Dcassandra.jmx.remote.port=7199
```

Comment:
Seemed like a bunch of good defaults.

Specifically excluded is the GC settings -- the operator should set this based on the container memory allocation.

Reply:
The operator should probably have exclusive control of setting jmx ports / configs, as the operator will always need to be able to do things to C* via JMX (e.g. decommission) unless of course, we shift that to the sidecar.

Reply:
Agree. I moved all the default configs back into the image because it seemed cleaner as the operator doesn't have to copy huge files and make a mess in a configmap for every deployment. Additionally it means that the image runs fine standalone, which helped me test a few things.

Having this baked-in with a known port number has no issues AFAIK -- but we could always drop in a jvm.options.d file with an explicit port.

Either way, not sure how we can totally prevent people from changing this port number since we allow custom jvm.options/env override. The result would be that the operator doesn't work and the fix is "don't do that".